python-avd/pyavd/_eos_designs/avdfacts.py

```diff
@@ -7,14 +7,14 @@
 from typing import TYPE_CHECKING, Any, Protocol
 
 if TYPE_CHECKING:
-    from collections.abc import Mapping
+    from collections.abc import MutableMapping
 
     from pyavd._eos_designs.schema import EosDesigns
     from pyavd._eos_designs.shared_utils import SharedUtilsProtocol
 
 
 class AvdFactsProtocol(Protocol):
-    _hostvars: Mapping
+    _hostvars: MutableMapping
```

Comment:
why MutableMapping instead of Mapping?

Reply:
This is because [here](https://github.com/aristanetworks/avd/blob/devel/python-avd/pyavd/api/interface_descriptions/__init__.py#L35) we use `ChainMap` which requires MutableMapping `class ChainMap(_collections_abc.MutableMapping):`